python/compile/compile_pip_requirements_wrapper_template.sh

```diff
@@ -10,15 +10,13 @@ PYTHON_INTERPRETER_PATH="@@PYTHON_INTERPRETER_PATH@@"
 PIP_COMPILE_BINARY="@@PIP_COMPILE_BINARY@@"
 
 echo "Compiling $REQUIREMENTS_TXT_PATH"
+
+export CUSTOM_COMPILE_COMMAND="@@CUSTOM_COMPILE_COMMAND@@"
 $PYTHON_INTERPRETER_PATH $PIP_COMPILE_BINARY \
     --output-file $REQUIREMENTS_TXT_PATH \
-    --no-header \
-    --no-index \
+    "@@QUIET_ARG@@" \
+    --no-emit-index-url \
     --generate-hashes \
     --allow-unsafe \
     "$@" \
     $REQUIREMENTS_IN_PATH
-set -x
-echo "@@HEADER@@" > requirements.tmp
-cat $REQUIREMENTS_TXT_PATH >> requirements.tmp
-cp requirements.tmp $REQUIREMENTS_TXT_PATH
```

Comment:
Hmm, default success output is _very_ quiet now, maybe just a success message would be good here
```console
$ bazel run //thirdparty/bazelbuild/pip/3:compile -- -P bcrypt 
INFO: Invocation ID: 98092195-083f-410a-bfe3-50c97d791aca
DEBUG: .bazel_out/external/bazel_tools/tools/osx/xcode_configure.bzl:243:14: No default Xcode version is set with 'xcode-select'; picking ':version15_2_0_15C500b'
INFO: Analyzed 2 targets (0 packages loaded, 0 targets configured).
INFO: Found 2 targets...
INFO: Elapsed time: 6.277s, Critical Path: 0.07s
INFO: 1 process: 1 internal.
INFO: Running command line: /bin/bash -c '/Users/ichamberlain/Documents/ssr/.bazel_out/execroot/com_128technology_i95/bazel-out/darwin-fastbuild/bin/tools/bazelbuild/util/rewrite_runfiles_paths bazel-bin/thirdparty/bazelbuild/pip/3/compile.sh --index-url https://artifactory.lab.128technology.com/api/pypi/pypi/simple -P bcrypt'
INFO: Build Event Protocol files produced successfully.
INFO: Build completed successfully, 1 total action
Compiling /Users/ichamberlain/Documents/ssr/thirdparty/bazelbuild/pip/3/requirements-osx.txt
    Couldn't get hashes from PyPI, fallback to hashing files
    Couldn't get hashes from PyPI, fallback to hashing files
$ 
```